docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2192,8 +2192,9 @@ Json Parameters:
 
 Status Codes:
 
--   **201** – no error
+-   **200** – no error
 -   **404** – no such exec instance
+-   **409** - Container is stopped or paused
```

Comment:
pre-docs-review ;-) s/Container/container/

can you add this to the API change log as well, e.g. "The /exec/[containerid]/start endpoint now returns a 409 if...". People may have implemented something that's expecting a 5xx error.